game-frontend/src/components/canvas-objects/Powerup.tsx

```diff
@@ -1,46 +1,55 @@
 import React, { FC } from 'react';
 import { Position } from '@darwin/types';
-import Circle from '../pixi/Circle';
+import { Sprite } from '@inlet/react-pixi';
 import { FIELD_SIZE } from '../../constants/stage';
-import Square from '../pixi/Square';
+import endurance from '../../assets/images/endurance.png';
+import teleport from '../../assets/images/teleport.png';
+import healthregen from '../../assets/images/healthregen.png';
 
 type Props = {
-  color: number;
   position: Position;
 };
 
-const HALF_FIELD_SIZE = FIELD_SIZE / 2;
-
-const squareScale = 0.4;
-const squareSize = FIELD_SIZE * squareScale;
-const squareMargin = (FIELD_SIZE - squareSize) / 2;
-
-const radius = HALF_FIELD_SIZE * 0.3;
+const HealthRegen: FC<Props> = ({ position }) => {
```

Comment:
What about adding the icon inside the colored circle? So it is better aligned with our overall art-style.
Also i would suggest to use uni-colored icons.

Reply:
i didn't manage to do this, maybe if you are familiar you can add the necessary change and i will take a look and learn :)